lib/events.js

```diff
@@ -52,11 +52,13 @@ Object.defineProperty(EventEmitter, 'defaultMaxListeners', {
   get: function() {
     return defaultMaxListeners;
   },
-  set: function(arg) {
+  set: function(n) {
     // force global console to be compiled.
     // see https://github.com/nodejs/node/issues/4467
     console;
-    defaultMaxListeners = arg;
+    if (typeof n !== 'number' || n < 0 || isNaN(n))
```

Comment:
@mscdex Is [this similar place](https://github.com/nodejs/node/pull/11938/files#diff-71dcd327d0ca067b490b22d677f81966R86) also worth replacing?